core/src/components/datetime/datetime.tsx

```diff
@@ -565,8 +565,7 @@ export class Datetime implements ComponentInterface {
   }
 
   private hasValue(): boolean {
-    const val = this.datetimeValue;
-    return Object.keys(val).length > 0;
```

Comment:
is this related to the change?

Reply:
Mm no. I branched off of master and am comparing to the hotfix. Must've accidentally pulled in an unrelated fix, good catch!